FileIO/GMSInterface.cpp

```diff
@@ -314,10 +319,23 @@ MeshLib::Mesh* GMSInterface::readGMS3DMMesh(const std::string &filename)
 	}
 
 	in.close();
-	INFO("GMSInterface::readGMS3DMMesh(): \tfinished.");
+	INFO("GMSInterface::readGMS3DMMesh(): finished.");
 
 	const std::string mesh_name = BaseLib::extractBaseNameWithoutExtension(filename);
-	return new MeshLib::Mesh(mesh_name, nodes, elements);
+	MeshLib::Properties properties;
+	if (mat_ids.size() == elements.size())
+	{
+		boost::optional<MeshLib::PropertyVector<int> &> opt_pv
+			= properties.createNewPropertyVector<int>(
+			"MaterialIDs", MeshLib::MeshItemType::Cell);
+		assert(opt_pv != boost::none);
```

Comment:
@rinkk What is the desired behavior if the `mat_ids` were read correctly (eg `mat_ids.size == elemenst.size`) but can not be copied to `opt_pv` because its creation failed?